<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>playerMove</class>
 <widget class="QDialog" name="playerMove">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextBrowser" name="leftDice">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QTextBrowser" name="rightDice">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>40</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelDiceLeft">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Wurf 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDiceRight">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Wurf 2</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="result">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="inputText">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>230</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelInput">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>240</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Eingabe</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelResult">
   <property name="geometry">
    <rect>
     <x>64</x>
     <y>140</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Dein Ergebnis</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCommit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>310</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Commit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonRoll">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>310</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Roll Dices</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCurrent">
   <property name="geometry">
    <rect>
     <x>64</x>
     <y>190</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Aktueller Stand</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="currentValue">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>180</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
